test/e2e/persistent_volumes.go

```diff
@@ -72,6 +72,19 @@ func completeMultiTest(f *framework.Framework, c clientset.Interface, ns string,
 	deletePVCandValidatePVGroup(c, ns, pvols, claims)
 }
 
+
+// initNFSserverPod wraps volumes.go's startVolumeServer to return a running nfs host pod
+// commonly used by persistent volume testing
+func initNFSserverPod(c clientset.Interface, ns string) *v1.Pod {
```

Comment:
I was thinking that pvutil.go would be common to all providers.  `initNFSserverPod` seemed a little too specific for it.